server/events/command/project_context.go

```diff
@@ -94,14 +94,13 @@ type ProjectContext struct {
 	ExecutionOrderGroup int
 }
 
-// SetScope sets the scope of the stats object field. Note: we deliberately set this on the value
-// instead of a pointer since we want scopes to mirror our function stack
-func (p ProjectContext) SetScope(scope string) {
+// SetScopeTags adds ProjectContext tags to a new returned scope.
+func (p ProjectContext) SetScopeTags(scope tally.Scope) tally.Scope {
```

Comment:
After a lot of debugging I got to the conclusion that the previous code was doing nothing, so instead of creating a function to `SetScope` that was basically `scope.SubScope`, I renamed to `SetScopeTags` that will add the listed tags to the scope